python/tests/test_highlevel.py

```diff
@@ -2213,12 +2266,21 @@ def test_newick_topology_equiv(self):
         replace_numeric = {ord(x): None for x in "1234567890:."}
         for ts in get_example_tree_sequences():
             for tree in ts.trees():
-                if tree.num_roots > 1:
-                    continue
-                plain_newick = tree.newick(node_labels={}, include_branch_lengths=False)
-                newick1 = tree.newick().translate(replace_numeric)
-                newick2 = tree.newick(node_labels={}).translate(replace_numeric)
-                assert newick1 == newick2 == plain_newick
+                if not tree.has_single_root:
+                    with pytest.raises(ValueError) as plain_newick_err:
+                        tree.newick(node_labels={}, include_branch_lengths=False)
+                    with pytest.raises(ValueError) as newick1_err:
+                        tree.newick().translate(replace_numeric)
```

Comment:
What's the point of the ``.translate`` here? This won't get called, right because newick() raises a ValueError?

Reply:
Well spotted. Copy-paste error. Thanks.